src/odemis/driver/xt_client.py

```diff
@@ -2228,6 +2277,10 @@ def _setBeamletIndex(self, beamlet_index):
         new_beamlet_index = self.parent.get_beamlet_index()
         return tuple(int(i) for i in new_beamlet_index)  # convert tuple values to integers.
 
+    def _setImmersion(self, immersion):
+        self.parent.set_compound_lens_focusing_mode(COMPOUND_LENS_FOCUS_IMMERSION if immersion else 0)
```

Comment:
Can you add a small comment to explain this `COMPOUND_LENS_FOCUS_IMMERSION if immersion else 0`?